inspectit-oce-documentation/src/docs/asciidoc/_includes/metrics/metrics-recorders.adoc

```diff
@@ -0,0 +1,305 @@
+=== Available Metrics
+
+Metrics recorders are responsible for capturing system metrics, such as processor or memory usage. All metrics recorders provided by inspectIT OCE publish the recorded data through the OpenCensus API.
+Therefore all recorded metrics can be exported via any of the provided metrics exporters.
+Currently the inspectIT OCE agent is capable of recording the following metrics:
+
+* <<Processor Metrics,Processor>> (usage and processor count)
+* <<Disk Space Metrics,Disk Space>> (used and total)
+* <<Memory Metrics,Memory>> (used and availalbe for various regions)
+* <<Thread Metrics,Threads>> (counts and states)
+* <<Garbage Collection Metrics,Garbage Collection>> (Pause times and collected amounts)
+* <<Class Loading Metrics,Class Loading>> (loaded and unloaded counts)
+
+[TIP]
+====
+The metrics and their capturing logic are based on the open-source https://micrometer.io/[micrometer] project.
```

Comment:
"The above metrics ..."

Later we will also have metrics that are not from micrometer.